vega_sim/network_service.py

```diff
@@ -228,6 +299,13 @@ def data_node_grpc_url(self) -> str:
             self._data_node_grpc_url = f"{url.split(':')[0]}:{DATA_NODE_GRPC_PORT}"
         return self._data_node_grpc_url
 
+    @property
+    def data_node_query_url(self) -> str:
+        if self._data_node_query_url is None:
+            url = self.network_config["API"]["GraphQL"]["Hosts"][1]
```

Comment:
I know we're updating this so probably no need to actually change here, but wouldn't this be safer as [0] for the case where there's only one?

Reply:
Definitely right, it's a legacy bit from when I needed a quick fix for node [0] being down!

It is replaced in the next PR.